src/assets/i18n/cz.json

```diff
@@ -189,6 +178,13 @@
 			},
 			"footer": "TO BE TRANSLATED - If you have any other problems or suggestions for improvement, you can send them to us via this form <a target=\"_blank\" href=\"https://www.cnil.fr/fr/webform/vos-retours-sur-loutil-pia\">https://www.cnil.fr/fr/webform/vos-retours-sur-loutil-pia</a>."
 		},
+		"infografics": {
+			"display": "Zobrazit mapu procesu",
+			"file": "./pia_docs/180409_fiche_risque_en_cmjk.pdf",
+			"guide": "Zobrazit průvodce PIA CNIL",
+			"hide": "Skrýt mapu procesu",
+			"title": "Přehled povinností a metody"
+		},
```

Comment:
Why this block?

Reply:
This is in preparation for the addition of the process map to help made in update 5.

Reply:
Ok thx, we need to review the Step 5 before.

Reply:
I've splitted the update in this way to reduce the number of files you have to review.

Otherwise the `i18n/*.json` files have to be reviewed in three steps in updates 3-5. So in update 4
- the date translations have been removed. They became obsolete by update 3.
- the translation additions for `help.infografics` and `cartography.risk3_disappearance` are part of update 4. The functionality itsself is added in update 5.